src/Pages/GlobalConfigurations/ClustersAndEnvironments/ClusterEnvironmentDrawer/ClusterEnvironmentDrawer.tsx

```diff
@@ -238,7 +238,7 @@ export const ClusterEnvironmentDrawer = ({
                     </button>
                 </div>
                 <form
-                    className="flex-grow-1 flexbox-col"
+                    className="flex-grow-1 flexbox-col dc__overflow-scroll mh-0"
```

Comment:
Why this `dc__overflow-scroll` is required?

Reply:
To scroll the data when overflowing